src/prover/calculi/Higher_order.ml

```diff
@@ -477,17 +477,15 @@ module Make(E : Env.S) : S with module Env = E = struct
     Util.incr_stat stat_ho_unif;
     let offset = C.Seq.vars c |> T.Seq.max_var |> succ in
     begin
-      HO_unif.unif_pairs ?fuel:None (pairs,0) ~offset
+      HO_unif.unif_pairs ?fuel:(Some 2) (pairs,0) ~offset
```

Comment:
this is good for exposing the bug, but not in general :-)
can you change this one back?

Reply:
oops